src/components/SarcophagusCreate/ArchaeologistSelect/ArchaeologistTableRow.tsx

```diff
@@ -39,11 +41,14 @@ const ArchaeolgistTableRow = ({
   const isDiggingFeeLess = archaeologist.minimumDiggingFee.lte(
     ethers.utils.parseEther(values.diggingFee.toString())
   );
-  const isDisabled = !isBountyLess || !isDiggingFeeLess;
+  const isArchaeologistMaxResValid = isMaxResurrectionTimeValid(archaeologist.maximumResurrectionTime.toNumber(), Number(values.resurrectionTime))
+
+  const isDisabled = !isBountyLess || !isDiggingFeeLess || !isArchaeologistMaxResValid;
   // checks freebond is greater
   const isFreeBondGreater = archaeologist.freeBond.gte(ethers.utils.parseEther(archTotal));
```

Comment:
@Da-Colon -- minor change for future branch -- group the following code together:

```
 const isFreeBondGreater = archaeologist.freeBond.gte(ethers.utils.parseEther(archTotal));
  if (!archaeologist?.isOnline || !isFreeBondGreater) {
    return null;
  }
```